src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AssessmentService.kt

```diff
@@ -571,7 +570,7 @@ class AssessmentService(
                   name = staffDetails.probationArea.description,
                 ),
                 cru = Cru(
-                  name = cruService.cruNameFromProbationAreaCode(staffDetails.probationArea.code),
+                  name = user.apArea?.name.toString(),
```

Comment:
I _think_ toString() is redundant here? (and everywhere else name is used in this PR)

Reply:
There's a type issue between 
apArea.name  (nullable)
cru.name (non-nullable)
toString() mitigates this but does mask the possibility of null getting through as 'null'
What's the desired behaviour?   
Would !! be more appropriate?

Reply:
Ah, good point!

This works because toString is an extension function that can operate on nulls. In this case it would return the string "null". See https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/to-string.html

user.apArea should _really_ be non-nullable as we always populate it, so the null scenario should never happen here. Regardless, i think it's probably more readable if we state what is returned when apArea is null. For consistency with what CruService was doing, maybe we could use this in all cases...

`user.apArea?.name ?: "Unknown CRU"`